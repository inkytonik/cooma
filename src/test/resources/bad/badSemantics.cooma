{fun (x : Int) = x}("hello")
