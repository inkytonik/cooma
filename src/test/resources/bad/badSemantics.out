src/test/resources/bad/badSemantics.cooma:1:19:error: expected Int, got "hello" of type String
{fun (x : Int) x}("hello")
                  ^
